lte/gateway/python/magma/pipelined/app/gy.py

```diff
@@ -37,25 +48,35 @@ class GYController(PolicyMixin, MagmaController):
     The GY controller installs flows for enforcement of GY final actions, this
     includes redirection and QoS(currently not supported)
     """
-
     APP_NAME = "gy"
     APP_TYPE = ControllerType.LOGICAL
+    ENFORCE_DROP_PRIORITY = flows.MINIMUM_PRIORITY + 1
+    # For allowing unlcassified flows for app/service type rules.
+    UNCLASSIFIED_ALLOW_PRIORITY = ENFORCE_DROP_PRIORITY + 1
+    # Should not overlap with the drop flow as drop matches all packets.
+    MIN_GY_PROGRAMMED_FLOW = UNCLASSIFIED_ALLOW_PRIORITY + 1
+    MAX_GY_PRIORITY = flows.MAXIMUM_PRIORITY
+    # Effectively range is 3 -> 65535
+    GY_PRIORITY_RANGE = MAX_GY_PRIORITY - MIN_GY_PROGRAMMED_FLOW
 
     def __init__(self, *args, **kwargs):
         super(GYController, self).__init__(*args, **kwargs)
+        self._config = kwargs['config']
         self.tbl_num = self._service_manager.get_table_num(self.APP_NAME)
         self.next_main_table = self._service_manager.get_next_table_num(
             self.APP_NAME)
+        self._enforcement_stats_scratch = self._service_manager.get_table_num(
```

Comment:
`_enforcement_stats_scratch` -> `_enforcement_stats_tbl`

Its actually a regular table so lets use proper naming, I noticed that I forgot to refactor enf controller(I'll refactor it there)